pkg/api/v1/types.go

```diff
@@ -684,6 +685,24 @@ type GCEPersistentDiskVolumeSource struct {
 	ReadOnly bool `json:"readOnly,omitempty" protobuf:"varint,4,opt,name=readOnly"`
 }
 
+// Represents a Quobyte mount that lasts the lifetime of a pod.
+// Quobyte volumes do not support ownership management or SELinux relabeling.
+type QuobyteVolumeSource struct {
+	// Registry represents a single or multiple Quobyte Registry services
+	// which acts as the central registry for volumes
+	Registry string `json:"registry" protobuf:"bytes,1,opt,name=registry"`
+
+	// Volume is the Quobyte volume name.
```

Comment:
It's a string and it references to an already created Quobyte Volume. So the user basically copies the name from the Quobyte Registry and set it in the definition file of the Volume.